app/views/items/index.html.erb

```diff
@@ -2,7 +2,11 @@
   <div class="jumbotron, index-jumbotron">
     <div class="jumbotron-contents">
       <h1 class="text-center, text-in-jumbotron"> Items for Sale </h1>
-      <%= button_to "Add Item", items_new_path, method: :get, class: "btn btn-success, btn-add-item" %>
+      <% if current_user %>
+        <%= button_to "Add Item", items_new_path, method: :get, class: "btn btn-success, btn-add-item" %>
```

Comment:
A small detail here, I don't think you need the node classes to be comma separated white space is enough